ci/pipeline.yml

```diff
@@ -477,6 +512,51 @@ resources:
     access_token: {{status_access_token}}
     every: true
 
+- name: terraform-yaml-tooling
+  type: s3
+  source:
+    bucket: {{aws_s3_tfstate_bucket}}
+    versioned_file: {{tf_state_file_tooling}}
+    access_key_id: {{aws_access_key_id}}
+    secret_access_key: {{aws_secret_access_key}}
+    region_name: {{aws_default_region}}
+
+- name: terraform-yaml-staging
+  type: s3
+  source:
+    bucket: {{aws_s3_tfstate_bucket}}
+    versioned_file: {{tf_state_file_staging}}
+    access_key_id: {{aws_access_key_id}}
+    secret_access_key: {{aws_secret_access_key}}
+    region_name: {{aws_default_region}}
+
+- name: terraform-yaml-production
+  type: s3
+  source:
+    bucket: {{aws_s3_tfstate_bucket}}
+    versioned_file: {{tf_state_file_production}}
+    access_key_id: {{aws_access_key_id}}
+    secret_access_key: {{aws_secret_access_key}}
+    region_name: {{aws_default_region}}
+
+- name: terraform-yaml-external-staging
+  type: s3
+  source:
+    bucket: {{aws_external_s3_tfstate_bucket}}
+    versioned_file: {{tf_state_file_external_staging}}
+    access_key_id: {{aws_external_access_key_id}}
+    secret_access_key: {{aws_external_secret_access_key}}
+    region_name: {{aws_external_region}}
+
+- name: terraform-yaml-external-production
+  type: s3
```

Comment:
For some reason, I only let the concourse role write to one file: https://github.com/18F/cg-provision/blob/master/terraform/modules/iam_role/concourse_worker/role.tf#L40-L48